hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java

```diff
@@ -347,6 +347,13 @@ private int getWhenNoSSLDetected() {
     public static final HazelcastProperty OPERATION_CALL_TIMEOUT_MILLIS
             = new HazelcastProperty("hazelcast.operation.call.timeout.millis", 60000, MILLISECONDS);
 
+    /**
+     * Timeout in seconds, used to wait for the proxy destroy operations
+     * (see. {@link com.hazelcast.spi.impl.proxyservice.ProxyService#destroyDistributedObject(String, String)} to finish.
+     */
+    public static final HazelcastProperty PROXY_DESTROY_TIMEOUT_SECONDS
```

Comment:
Do we need this property in the public API? The only place where it is required is `LocalIndexStatsTest`. If there are no demand for this from users, I would rather hide it inside a private API class. Or even set it through a setter method of a `ProxyServiceImpl` after the node started.

Reply:
Also, https://hazelcast.atlassian.net/wiki/spaces/EN/pages/253953578/Hazelcast+Configuration#HazelcastConfiguration-CritiqueofGroupProperties

Reply:
I also thought the same but others were faster. To contribute something original to the debate, the javadoc of this property (if it stays in the public API) can be improved, namely the unclosed `(see...` part that does not parse for me.

Reply:
I listened to your demands, and I made it happen! its a private property now. I agree its very limited use-case so it doesn't even worth the private API, the private prop is a better fit.